services/web/README.md

```diff
@@ -24,3 +24,113 @@ When defined via the `WEB_UI_CONFIG_FILE` environment variable, the configuratio
 ### Embedding Web
 
 Web can be consumed by another application in a stripped down version called “Embed mode”. This mode is supposed to be used in the context of selecting or sharing resources. For more details see the developer documentation [ownCloud Web / Embed Mode](https://owncloud.dev/clients/web/embed-mode/). See the environment variables: `WEB_OPTION_MODE` and `WEB_OPTION_EMBED_TARGET` to configure the embedded mode.
+
+# Web Apps
+
+The administrator of the environment is capable of providing custom web applications to the users.
+This feature is useful for organizations that want to provide third party or custom apps to their users.
+
+It's important to note that the feature at the moment is only capable of providing static (js, mjs, e.g.) web applications
+and does not support injection of dynamic web applications (custom dynamic backends).
+
+## Loading Applications
+
+Loading applications is achieved in three ways:
+
+* By loading a web application from a user-provided path, by setting the `WEB_ASSET_APPS_PATH` environment variable.
+* By loading a web application from the default ocis home directory, e.g. `$OCIS_BASE_DATA_PATH/web/assets/apps`.
```

Comment:
While the first sentence is clear as you define a path in a variable, the second sentence is not clear.
The value provided is a path but where to configure it?

Reply:
$OCIS_BASE_DATA_PATH is our default ocis home path, i have no idea how to describe it better, can you provide a better explaining version?

Reply:
We have solved this via chat and you commit added